tools/ld-dropout-correct/correctorpool.cpp

```diff
@@ -44,32 +44,32 @@ bool CorrectorPool::process()
         if (!targetVideo.open(stdout, QIODevice::WriteOnly)) {
                 // Could not open stdout
                 qInfo() << "Unable to open stdout";
-                sourceVideos[0].close();
```

Comment:
Closing *one* of the source videos doesn't sound right (and similarly below). It would probably be better for the caller to be responsible for closing the videos, since it provided them in the first place.

Reply:
Done